client/src/EmailLinkLoginPage.js

```diff
@@ -25,6 +24,14 @@ class EmailLinkLoginPage extends Component {
     this.authenticate = this.authenticate.bind(this);
   }
 
+  componentWillMount() {
+    document.body.style.backgroundColor = "white";
+  }
+
+  componentWillUnmount() {
+    document.body.style.backgroundColor = null;
+  }
```

Comment:
So my intuition with changing the background colors was to change it in the css file. However, it seems that setting the background of the outermost div in the render method just gives us a white box on a black background.
<img width="720" alt="screen shot 2018-03-27 at 8 56 26 pm" src="https://user-images.githubusercontent.com/11862658/38002631-6cc34c7a-3201-11e8-9300-574453eea56d.png">

I get the right behavior if I move the 
```
document.body.style.backgroundColor = "white"
```
to the render method but I'm not sure if that solves the problem. My understanding is that you're saying my fix might not work for mobile? If I do keep my workaround, how do I factor out the color change into a new component and add it back in? I'm not quite following your explanation.

Reply:
Got it!  Yeah, looking at CSS is a good intuition as to where to start.  There's some complexity here in that the CSS is written directly into the HTML (see link above) to always apply to the body tag, since the app was initially setup to render in the full screen on mobile, but to be styled to look the size of a mobile app if you're on desktop.  That's why there's no obvious way to change that.

So to change that code, you'll have to test across mobile and desktop screen sizes and make sure things still work.  With this work, it'd be great to avoid that so your workaround strategy is great. 👍 

My suggestion here is that you add a comment about this, and since it's kind of a weird quirk and you've had to add it in multiple places here already (and will for other pages that you want to have all white background), then to factor it into it's only tiny component that just sets this style and explains why it's doing that in a comment.  Alternately, you could work around and keep the black background styling (that's what [this page](https://teachermoments.teachingsystemslab.org/equity) does).

Reply:
gotcha. How would I go about factoring this styling quirk into its own separate component? Should I just make a new js file in the research folder, import the component in LoginPage.js and EmailLinkLoginPage.js, and render the component? I think I'm a bit lost on how this happens with the React components

Reply:
Ah cool this is a good time to learn then!  :)  Yep, it's just like all the other components you've been working with and writing, you just have to make the new file and import like you described.

Reply:
done!